br.app.pw3270.terminal.yml

```diff
@@ -0,0 +1,43 @@
+app-id: br.app.pw3270.terminal
+runtime: org.gnome.Platform
+runtime-version: 43
+sdk: org.gnome.Sdk
+command: pw3270
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--own-name=br.app.pw3270.pw3270.terminal"
```

Comment:
based on the app-id `br.app.pw3270.terminal`  would be grantd by default.

The one above would be granted automatically if  the app id was `br.app.pw3270.pw3270`